requirements.txt

```diff
@@ -1,4 +1,5 @@
 pillow>=5.3,<6
 scipy>=1.3,<2
 tensorflow>=2,<3
-tqdm>=4,<5
+keras
```

Comment:
should we fix the major version at least?

Reply:
Was relying on dependabot :D... also presumably installing at the same time as tensorflow will ensure compatible versions

Reply:
👍